bld/CLMBuildNamelist.pm

```diff
@@ -157,6 +158,11 @@ OPTIONS
      -dynamic_vegetation      Toggle for dynamic vegetation model. (default is off)
                               (can ONLY be turned on when BGC type is 'cn' or 'bgc')
                               This turns on the namelist variable: use_cndv
+     -fan "mode"              Set how and if FAN is run: atm|soil|full|on|off
+                              If "on", FAN is enabled but not connected to atmosphere or
+                              soil biogeochemistry. If "full", both connections are
+                              active. The soil and atmosphere coupling can be enabled
+                              selectively with the "soil" and "atm" modes.
```

Comment:
This makes sense to me. @ekluzek how are multiple flags like this handled elsewhere in the code?

Reply:
Also, should we make it clear here, or elsewhere in the code that when 'soil' is active we're really just talking about soil in crop columns, as FAN manure inputs are put onto non-crop columns, but these fluxes are not considered in the current implementation?

Reply:
@ekluzek I think we have discussed moving away from having namelist opts like this, because they're confusing for the user - instead using just namelist variables where possible, or xml variables where we need something that controls a number of other things. Do you agree?

Reply:
There is actually a namelist option to enable FAN fluxes also for non-crops, but it is off by default.

Reply:
@juliusvira the point is that simple namelist options shouldn't be added as a command line option to build-namelist. The command line options are for things that are complicated and involve many namelist items. And as @billsacks points out if needed a xml option could also be added. But, in this case it looks like there could be a single simple namelist option that can control FAN. So it's something that can just be turned on by adding it to the user_nl_clm file. I think that's a better user interface for FAN than through the build-namelist command line.